templates/python36build.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
Looks like this is missing recent template changes in a diff against pythonbuild.json -- addition of lookupPolicy